library/core/src/char/convert.rs

```diff
@@ -199,6 +199,8 @@ impl const From<u8> for char {
 }
 
 /// An error which can be returned when parsing a char.
+///
+/// This `struct` is created when using the [`char::from_str`](char#tymethod.from_str) method.
```

Comment:
I think it's still not ready:

```
warning: unresolved link to `char::from_str`
 --> foo.rs:1:7
  |
1 | /// [`char::from_str`]
  |       ^^^^^^^^^^^^^^ the builtin type `char` has no associated item named `from_str`
```

We need https://github.com/rust-lang/rust/issues/90703 to be fixed first.

Reply:
Wait, this doesn't match the error above though. The CI failure resolved the link correctly, it's just a dead link for some reason. What version of rustdoc did you use to test locally? Does the error change if you use beta?

Reply:
Oh, it's because you used a fragment link instead of an intra-doc link.

Reply:
Yes, which is invalid too. I'll update the current code but even the fixed version is broken for the moment.